.github/workflows/cmake.yml

```diff
@@ -45,8 +45,8 @@ jobs:
           cmake-linux-ci-opts-
     - name: Compile
       run: |
-        cmake --preset linux-ci-build-with-nonstandard-options
-        cmake --build --preset linux-ci-build-with-nonstandard-options 
+        cmake --preset linux-ci-build-with-nonstandard-options -DQL_COMPILE_WARNING_AS_ERROR=ON -L
```

Comment:
In this case we're already calling a preset, so we can put the `QL_COMPILE_WARNING_AS_ERROR=ON` in there.  Same in another case below.

Reply:
Sure, makes sense. I'll incorporate it.